cranelift/filetests/filetests/isa/x86/windows_fastcall_x64_unwind.clif

```diff
@@ -118,6 +118,48 @@ block0:
 ; nextln:     ],
 ; nextln: }
 
+function %fpr_with_function_call(i64, i64) windows_fastcall {
+    fn0 = %foo(f64, f64, i64, i64, i64) windows_fastcall;
+block0(v0: i64, v1: i64):
+    v2 = load.f64 v0+0
+    v3 = load.f64 v0+8
+    v4 = load.i64 v0+16
+    v15 = load.f64 v0+104
+    v16 = load.f64 v0+112
+    v17 = load.f64 v0+120
+    v18 = load.f64 v0+128
+    v19 = load.f64 v0+136
+    v20 = load.f64 v0+144
+    v21 = load.f64 v0+152
+    v22 = load.f64 v0+160
+    v23 = load.f64 v0+168
+    call fn0(v2, v3, v4, v1, v1)
+    store.f64 v15, v1+104
+    store.f64 v16, v1+112
+    store.f64 v17, v1+120
+    store.f64 v18, v1+128
+    store.f64 v19, v1+136
+    store.f64 v20, v1+144
+    store.f64 v21, v1+152
+    store.f64 v22, v1+160
+    store.f64 v23, v1+168
+    return
+}
+; sameln: UnwindInfo {
+; nextln:     version: 1,
+; nextln:     flags: 0,
+; nextln:     prologue_size: 26,
+; nextln:     unwind_code_count_raw: 7,
+; nextln:     frame_register: 5,
+; nextln:     frame_register_offset: 12,
+; nextln:     unwind_codes: [
+; nextln:       UnwindCode {
+; nextln:         offset: 26,
+; nextln:         op: SaveXmm128,
+; nextln:         info: 15,
+; nextln:         value: U16(
+; nextln:             3,
+
```

Comment:
This seems incomplete.

Reply:
I was only particularly concerned with verifying that `xmm15` ends up with an offset that doesn't overlap with shadow space or stack argument space for the called function. I can include the rest of the function's `UnwindCode` if you'd like, but I figured that got outside the point of this test.

Reply:
I'm good with this but perhaps with a comment explaining why only the first unwind code is validated?

Reply:
Done - [see here](https://github.com/bytecodealliance/wasmtime/pull/1216/files#diff-f237b61b6e0ae8ac7e42cc2e94fbe7e1R148-R151)